src/graph/serializers/decoders/prev/decode_previous.c

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2018-2019 Redis Labs Ltd. and Contributors
+ *
+ * This file is available under the Redis Labs Source Available License Agreement
+ */
+
+#include "decode_previous.h"
+#include "../../../../version.h"
+
+GraphContext *Decode_Previous(RedisModuleIO *rdb, int encver) {
+	switch(encver) {
+	case 4:
+		return RdbLoadGraphContext_v4(rdb);
+	case 5:
+		return RdbLoadGraphContext_v5(rdb);
+	default:
+		assert(false);
```

Comment:
can we add feedback? maybe in the runtime error branch?

Reply:
This isn't a user-reachable spot (and should never be if we keep our deserializers up-to-date). The user-facing error messages are captured in the caller, `GraphContextType_RdbLoad`, such as:
```
Failed loading Graph, RedisGraph version (%d) is not backward compatible with encoder version %d
```